systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m

```diff
@@ -2140,31 +2148,26 @@ function checkDirty(obj)
 
     function fr = constructInputFrame(model)
       %inputparents is an array of the parent RigidBodies of each joint.
```

Comment:
this is the most significant change that could impact existing code. 
pretty sure I got it correct, but extra eyes would be appreciated.